piccolo_api/crud/endpoints.py

```diff
@@ -315,7 +314,7 @@ def _pydantic_model_output(
             include_readable=include_readable,
             include_columns=include_columns,
             model_name=f"{self.table.__name__}Output",
-            nested=nested,
+            nested=nested,  # type:ignore
```

Comment:
What's the type error here?

Reply:
OK, I see the problem. I think it has always been an issue, but before we wrapped `create_pydantic_model` in `@lru_cache`, which probably meant mypy couldn't see it. I have a fix.